.github/workflows/go.yml

```
@@ -48,8 +48,16 @@ jobs:
       uses: golangci/golangci-lint-action@v8
       if: runner.os == 'Linux'
       with:
```

```suggestion
 run: |
          if [ "$(gofmt -l . | wc -l)" -gt 0 ]; then
            echo "Formatting issues found:"
            gofmt -d .
            exit 1
          fi
        timeout-minutes: 30
```
    Try to remove lines 50-53 and replace with the above

---

@yasminvalim ^

---

see https://github.com/coreos/butane/actions/runs/14868423361/workflow